.gitignore

```diff
@@ -213,6 +216,12 @@ vda/ssim
 vda/vda
 vda/vdad
 /win_build/*.sdf
+/win_build/*.xml
```

Comment:
`/win_build/` should not be used as a DATA directory

Reply:
You compile the boic.exe to win_build. 
When you debug it will create data file to there.
So this is the default folder as a data.
Developer should not see this extra files in the git when he push PR.
So if it is not data folder as you say, you shouldn't add xml file in a default. If want in the future to add an xml to win_build with other reason, you can still do it with git add force.
Then you know what are you doing.

Reply:
Just pass `datadir` parameter with value as a command line parameter to BOINC Manager or `dir` with value to BOINC client

Reply:
I don't think is a good idea that every time I start a new debug I should set something.
When I push play on the debug, it should work.
Maybe to create a default folder inside boinc for data, then change it for release. 
Not sure how to do it.

Reply:
You can set commandline parameters in this window (if you're using VS) and it will be saved
![image](https://user-images.githubusercontent.com/3234209/82140560-cbe6b200-982f-11ea-8cab-f5c1afa29e52.png)

Reply:
> You can set commandline parameters in this window (if you're using VS) and it will be saved

This is the things I am trying to avoid. Setting every time I am switching or change something in the branch.

Reply:
This setting will be saved even when you switch branches

Reply:
If you don't agree with me, you can close this PR.
But this is what is how you should work, easy and without any other settings.
If you can do it without intefeare the users  and other developer and make life easier then you can do it.